manifests/params.pp

```diff
@@ -20,6 +20,26 @@
     'Debian' => 'apt',
     default  => 'archive'
   }
+
+  if $facts['os']['name'] == 'Ubuntu' {
+    $duply_paramiko_package_name = versioncmp($facts['os']['release']['major'], '20.04') >= 0 ? {
```

Comment:
Enabling backups of 20.04 systems would be of interest to me too. @punycode is there any help I can provide in getting this included?